### PR TITLE
Update zc-conformance.md for accuracy and consistency

### PR DESCRIPTION
**Pull request by balagansky-work**
Changes assume accuracy of the individual option doc pages

* Update default values per individual option docs
* Consistent implied by wording and formatting
* Mention [-] syntax where applicable

### CODE REVIEW

**Comment by balagansky-work** [comment]
> @balagansky-work
> 
> Thank you for your contribution. Would you take a moment to sign the [Contributor License Agreement (CLA)](https://github.com/MicrosoftDocs/cpp-docs/pull/4665#issuecomment-1660525742)? After the CLA is signed, someone can review your pull request. Thanks!
> 
> #label:"aq-pr-triaged" @MicrosoftDocs/public-repo-pr-review-team

Apologies for the delay. I'm still waiting on my company's approval to sign the CLA.

File: docs/build/reference/zc-conformance.md
Comment (TylerMSFT) [inline docs/build/reference/zc-conformance.md:29]: Hi, I sometimes like the 'implied by' approach, but it doesn't mean the same thing as the text it replaces. It indicates that the switch may be on when another switch is specified, but it loses the clarity that it is off in all other cases.  It replaces a statement that is true in all cases with a more ambiguous one that it is true in a specific case. I prefer the wording for /Zc:char8_t, for example. I'd rather not take the 'implied by' changes unless they add the extra clarity that they are off, otherwise.  Thank you for adding the optional [-] notes!

Comment (balagansky-work) [reply docs/build/reference/zc-conformance.md:29]: My goal was to make these parentheticals consistent, which improves readability. I chose the "implied by" form because it was terser. But I don't mind changing it over to the form on this line.

Some cases are a bit different than others. If you still don't like the wording, let me know and I'll update it.

Comment (balagansky-work) [reply docs/build/reference/zc-conformance.md]: I replaced "implied by" everywhere with the form on line 29.

I also made another tweak to the twoPhase row. That one is a trickier special case. I hope the new wording is clearer. I think the original wording was prone to misinterpretation.

**Review by TylerMSFT — APPROVED** [review]
I like what you did there. Thank you!

**Comment by TylerMSFT** [comment]
#sign-off

**Comment by Court72** [comment]
> Apologies for the delay. I'm still waiting on my company's approval to sign the CLA.

@balagansky-work - No problem! After you've signed the CLA, this PR can be merged. Thanks!

#hold-off

**Comment by Court72** [comment]
@balagansky-work - Thanks for letting us know. The CLA completion is required for this PR. 

@TylerMSFT - Would you like to pull these changes into the private repo and merge?

**Comment by Court72** [comment]
@orcmid - That's correct! From the documentation @balagansky-work linked: "When your PR is created, it is classified by a CLA bot. If the change is trivial (for example, you fixed a typo), then the PR is labeled with CLA-not-required. Otherwise, it's classified as CLA-required. Once you signed the CLA, the current and all future pull requests are labeled as CLA-signed."

**Comment by TylerMSFT** [comment]
I don't want to lose the work that @balagansky-work has put in here.  I'll port the changes and check in under my account.
@balagansky-work, I'll let you know once that's done.

**Comment by TylerMSFT** [comment]
@balagansky-work, I have taken your changes and put them in a private repo and integrated them into the docs. They'll be live by tomorrow. 
The PR is https://github.com/MicrosoftDocs/cpp-docs-pr/pull/5022 
Thank you for contributing to the docs!

**Pull request closed without merging**
